### DISCUSSION TITLE
/meeting page bug

### DISCUSSION THREAD

**Discussion by nataliefaith2** [code4recovery/12-step-meeting-list#976] [Q&A] [answered]
Hi there,

I'm having issues with the /meeting page or adding the meeting feed with map to any other page using [tsmi-ui]

See the attached screenshot of the /meeting page (I can't share a link as this website is currently being built) - you can see a feed above the search-filter-map feed (which is the one I only want visible), and there's a meeting region links underneath it.

<img width="791" alt="Screen Shot 2022-11-20 at 12 32 44 pm" src="https://user-images.githubusercontent.com/23748349/202878539-64aa884a-5f53-437d-ab6f-1a4fed027fd3.png">

I should mention that I have connected the Google API to the plugin but it hasn't worked as the map is coming up with error messages. I saw another feed that both google API and Mapbox together makes it work - is that what you would advise?

I also want to separate the region feeds so those looking for meeting in those areas can go to a specific feed for this information and see where their closest location is, and have this visible on a map of that region - is that possible?

And if possible can I use the search, filter section on a page and like WP search the results below would show the search parameters - showing meeting content only?

**Comment by 97eighty** [comment, marked as answer]
Hi @nataliefaith2 Can you open the developer console and see if there are any errors from the map API? If you're developing this using localhost it could be a permissions problem. Are you using the [TSML-UI] shortcake on its own page? We could do a bit more troubleshooting if you could install this on a public address.
